### PR TITLE
[GH-167][GH-215] Removed "Beta" label on Account-Level app, and updated documentation

### LINKED ISSUE #167: Add screenshot to the README

### LINKED ISSUE #215: Document steps to authorize "Account Level App"
A customer was following the docs and ran into an issue with Zoom.  

They are using the  "Account Level App" integration (versus the "User Level").

We need to describe the authorization steps that a Zoom admin needs to follow to complete the integration.  This [issue comment](https://github.com/mattermost/mattermost-plugin-zoom/issues/203#issuecomment-745348681) explains the additional steps:

* On Mattermost, as a sysadmin, run /zoom connect
* Follow the link and follow the instructions

Not sure of the best place to organize this.  Offhand I'd say one of these approaches:

1. An additional "Account Level App"-specific instruction under the "Mattermost Setup" page.
2. Under "Usage" section have a "Connect your account (User Level)", and separate "Connect your account (Account level)"

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
#### Summary
1. Added the latest screenshot.
2. Completed the documentation for plugin configuration.

#### Screenshot:
image

#### Ticket Link
Fixes #167 
Fixes #215

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,12 +198,11 @@ Select **Done** and then save your app.
     * If you're using a self-hosted private cloud or on-premises Zoom server, enter the **Zoom URL** and **Zoom API URL** for the Zoom server, for example `https://yourzoom.com` and `https://api.yourzoom.com/v2` respectively. Leave blank if you're using Zoom's vendor-hosted SaaS service.
   * **Cloud Hosted?**
     * Leave **Zoom API URL** and **Zoom URL** fields blank.
+* If you are using an account level app on Zoom, set **OAuth by Account Level App (Beta)** to `true`.
```
Comment (mickmister) [inline README.md:201]: We should probably get rid of references to "Beta" on this feature at this point, including in the plugin setting definition

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
LGTM, just a few suggestions

File: README.md [hunk 2]
```diff
@@ -198,12 +198,11 @@ Select **Done** and then save your app.
...
 * Connect your users to Zoom using OAuth.
-  * Use the Client ID and Client Secret generated when configuring Zoom to fill in the fields **Zoom OAuth Client ID** and **Zoom OAuth Client Secret**.
+  * Use the Client ID and Client Secret generated when configuring Zoom to fill in the fields **Zoom OAuth Client ID** and **Zoom OAuth Client Secret**. (Make sure these credentials are according to the app type (user/account level) you are using.)
```
[the remark this answers is not in the document]
Comment (raghavaggarwal2308) [reply README.md:203]: @mickmister By this line I meant that if the user has selected the app type as "Account level app". He should make sure that he is using a account level app on Zoom side as well.

[the remark this answers is not in the document]
Comment (mickmister) [reply README.md:203]: Maybe we should just say the sentence that you used above to explain the point. If the "credentials" don't change between the two app types, then the current sentence could be misleading

File: README.md [hunk 3]
```diff
@@ -198,11 +198,11 @@ Select **Done** and then save your app.
     * If you're using a self-hosted private cloud or on-premises Zoom server, enter the **Zoom URL** and **Zoom API URL** for the Zoom server, for example `https://yourzoom.com` and `https://api.yourzoom.com/v2` respectively. Leave blank if you're using Zoom's vendor-hosted SaaS service.
   * **Cloud Hosted?**
     * Leave **Zoom API URL** and **Zoom URL** fields blank.
-* If you are using an account level app on Zoom, set **OAuth by Account Level App (Beta)** to `true`.
+* If you are using an account level app on Zoom, set **OAuth by Account Level App** to `true`.
 * Connect your users to Zoom using OAuth.
-  * Use the Client ID and Client Secret generated when configuring Zoom to fill in the fields **Zoom OAuth Client ID** and **Zoom OAuth Client Secret**. (Make sure these credentials are according to the app type (user/account level) you are using.)
+  * Use the Client ID and Client Secret generated when configuring Zoom to fill in the fields **Zoom OAuth Client ID** and **Zoom OAuth Client Secret**. (If you have selected the app type as "Account level app", make sure that you are using a account level app on Zoom side as well)
```
Comment (mickmister) [inline README.md:203]:
```suggestion
  * Use the Client ID and Client Secret generated when configuring Zoom to fill in the fields **Zoom OAuth Client ID** and **Zoom OAuth Client Secret**. (If you have selected the app type as "Account level app", make sure that you are using an account-level app on Zoom side as well)
```

**Review by mickmister — APPROVED** [review]
LGTM, just one grammatical suggestion

**Comment by raghavaggarwal2308** [comment]
@mickmister Fixed

**Review by mickmister — APPROVED** [review]

**Pull request merged**